modules/ocf/templates/firefox/prefs.js.erb

```diff
@@ -1,3 +1,25 @@
+// This is the Debian specific preferences file for Firefox ESR
+// You can make any change in here, it is the purpose of this file.
+// You can, with this file and all files present in the
+// /etc/firefox-esr directory, override any preference you can see in
+// about:config.
+//
+// Note that pref("name", value, locked) is allowed in these
+// preferences files if you don't want users to be able to override
+// some preferences.
```

Comment:
Even though Debian suggested these changes, I don't think there's any benefit in differentiating between these and "OCF prefrences." Since everything is hand-picked, they are all "OCF changes"